src/Provider/Youtube/VideoInfo.php

```diff
@@ -162,13 +162,13 @@ class VideoInfo implements VideoInfoInterface, CacheAware, HttpClientAware, Logg
     /**
      * Creates a VideoInfo from string with an options array
      *
-     * @param string $video_info
+     * @param string $video_id
      * @param array  $options
      * @param mixed  $string
      *
      * @return VideoInfo
      */
-    public static function createFromStringWithOptions($string, array $options)
+    public static function createFromStringWithOptions($string, array $options, $video_id)
```

Comment:
This new 3rd parameter must either have a default value or the tests must be changed to reflect this change. All travis builds failed because of this.
https://travis-ci.org/jeckman/YouTube-Downloader/builds/600005207